dependencies.yaml

```diff
@@ -56,32 +56,57 @@ files:
     includes:
       - docs
       - py_version
-  py_rapids_build_ucxx:
+  py_build_libucxx:
     output: pyproject
-    pyproject_dir: python/ucxx
+    pyproject_dir: python/libucxx
     extras:
       table: build-system
     includes:
       - rapids_build_skbuild
+  py_rapids_build_libucxx:
+    output: pyproject
+    pyproject_dir: python/libucxx
+    extras:
+      table: tool.rapids-build-backend
+      key: requires
+    includes:
+      - build_cpp
+      - depends_on_librmm
```

Comment:
I was going to say that this is introducing a hard dependency on RMM, but it's already the case in conda packages. Ideally we would have one that doesn't depend on RMM for non-CUDA users but we will need to resolve that for both conda and wheels in the future.

Reply:
Wouldn't that involve creating a package like `libucxx-nocuda` or something?

Reply:
Yes, either that or doing what UCX does and `dlopen` resources, although RMM is header-only so we may need to do some other runtime checks instead, and this is why it was done yet.